[AGM-1944] update AGVPN for iOS section (#161)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-ios/solving-problems/logs.md

## Collecting and sending standard logs

By default, AdGuard VPN for iOS uses the *Default* logging level, that is, the basic collection of data about running processes of the app. To send these logs, follow the next steps:

1. Connect to AdGuard VPN for iOS and, if possible, repeat the actions that led to the error. Note the exact time when the error occurred (e.g. 15:35).

2. Open AdGuard VPN and tap *Settings* ⚙ icon in the lower right corner of the screen, then tap *Support* and *Report a bug*.

3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.

4. Tick the *Send app logs and system info* checkbox, which means that when you send a report your logs will be sent along with it.

> 5. Press *Send* button.

Reviewer: ```suggestion
5. Tap *Send*.
```

> >If for some reason it is more convenient for you to send us logs in another way, you can export them yourself.

Reviewer: оформление некорректное

Reviewer: надо вот эти приколы с :::info

Reviewer: или note...

>To do this, go to *Settings* → *General* → *Advanced* and tap *Export logs and system info*.

## Collecting and sending detailed logs

In most cases, the default logging level is sufficient to trace down possible bugs. But there are cases when more detailed technical information about the device and connections is required, and then our support team will ask you to enable the *Extended* or *Extreme* logging level. To send this type of logs, follow these steps:

1. Open AdGuard VPN for iOS and tap *Settings* ⚙ icon in the lower right corner of the screen.

2. Select *General* and then *Advanced*.

3. In the *Logging level* section, select *Extended* or *Extreme*.

> 4. Repeat the actions that led to the error, and note the time when it occurred (e.g. 14:23).

Reviewer: ```suggestion
4. Repeat the actions that led to the error and note the time when it occurred (e.g. 14:23).
```

5. Switch the logging level back to *Default*.

> 6. Go back to *Settings* screen, tap *Support* and then *Report a bug*.

Reviewer: ```suggestion
6. Go back to *Settings*, tap *Support* and then *Report a bug*.
```

7. In the opened form, enter your email address for feedback and describe the error found, including the exact time when this error occurred.

> 8. Make sure that *Send app logs and system info* checkbox is ticked and press *Send* button.

Reviewer: ```suggestion
8. Make sure that *Send app logs and system info* is checked and tap *Send*.
```

> >If for some reason it is more convenient for you to send us logs in another way, you can export them yourself.

Author: не понял, какой тот же

> >If for some reason it is more convenient for you to send us logs in another way, you can export them yourself.

Author: а, все, понял
